django/contrib/admin/options.py

```diff
@@ -886,13 +882,10 @@ def get_actions(self, request):
         # If self.actions is set to None that means actions are disabled on
         # this page.
         if self.actions is None or IS_POPUP_VAR in request.GET:
-            return OrderedDict()
+            return {}
         actions = self._filter_actions_by_permissions(request, self._get_base_actions())
-        # Convert the actions into an OrderedDict keyed by name.
-        return OrderedDict(
-            (name, (func, name, desc))
-            for func, name, desc in actions
-        )
+        # Convert the actions into a dictionary keyed by name.
```

Comment:
Maybe that comment could be removed as the dict comprehension seems somewhat obvious.